CONTRIBUTING.md

```diff
@@ -91,15 +91,54 @@ it will create a new folder inside `packages` with a simple boilerplate for your
 
 ### Release
-- If necessary, make and merge a PR to contain any last minute changelog entries or other housekeeping items
-- Make a PR that contains the file changes from `pnpm lerna version --no-push 7.0.0` (update the version as necessary)
-- When pushing the release PR, use `--follow-tags` to make sure the relevant tag is also pushed
-- The CI will start doing the release when the tag is created
-- Don't forget to merge your PR
+- If necessary, make and merge a PR with any last minute housekeeping items (docs, etc.)
+- Review code commits and decide the new version number.  If there are breaking changes, then it should be a major version bump, e.g. 6.x.x to 7.0.0.  This project follows [semantic versioning](https://semver.org/).
+
+#### Release commands
+Run the following release commands, replacing 7.0.0 with the version number to release:
+
+- fetch the latest code from remote origin
+```bash
+git fetch origin
+```
+
+- create a release branch, replace mf with your initials to make it clear whose branch it is.
+```bash
+get checkout origin/master -b mf/release-7.0.0
+```
+
+- increment the version number of all packages, without pushing to origin.  This will also create a release tag.
+```bash
+pnpm lerna version --no-push 7.0.0
+```
+
+- check the changed files and stage them for commit.  Example:
+```bash
+git add packages/*/package.json
+```
+
+- create new commit on the release branch
+```bash
+git commit -m "Release v7.0.0"
+```
+
+- push the release branch and the release tag.  This will trigger the Github [release](https://github.com/Turfjs/turf/blob/master/.github/workflows/release.yml) action.
+```bash
+git push origin mf/release-7.0.0 --follow-tags
+```
+
+#### Release Final Steps
+- Create a Pull Request for the release, using the link in the output of the push command and the version number for the PR title (e.g. v7.0.0).  If the link isn't provided, just create a PR setup to merge your release branch to master.  Don't merge the PR yet.  Here is an example - https://github.com/Turfjs/turf/pull/2615
+
+- You can view the status of the triggered release action here - https://github.com/Turfjs/turf/actions.  Once complete, a new [version](https://www.npmjs.com/package/@turf/turf?activeTab=versions) of all turf packages will have been published on NPM.
+
+- If NPM publish was successful, merge your release PR to master.
+- [Snapshot and release](#documentation) a new version of the API docs.
+- As part of the release action, a draft Github release will have been created at https://github.com/Turfjs/turf/releases with an auto-generated changelog.  Edit and add to the release notes for readability and completeness, specifically noting any breaking changes.  Use past releases as a guide.  Be sure to "Save draft" each time, then ask for a review from other contributors.  Once ready, click `Publish release`.  This will make the release notes publicly accessible and notify all watchers of the project.
 
 ## Documentation
 
```

Comment:
In "Other Dependencies" below (sorry, github won't let me add a comment on that line) where it says "yarn" it should say "pnpm"

Should be able to remove "Yarn will need to be installed on your computer ..." as pnpm is included in every node install by default.

Reply:
Done